src/uct/api/v2/uct_v2.h

```diff
@@ -671,7 +671,10 @@ typedef enum uct_md_attr_field {
     UCT_MD_ATTR_FIELD_EXPORTED_MKEY_PACKED_SIZE = UCS_BIT(13),
 
     /** Unique global identifier of the memory domain. */
-    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14)
+    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14),
+
+    /** Indicate memory types that are ODPable */
```

Comment:
Maybe rephrase to `Indicate memory types that the MD can register using ODP`, sounds more formal?
@brminich  WDYT?

Reply:
```suggestion
    /** Indicate memory types that the MD can register using ODP */
```

Reply:
WDYTA?
```
/** Indicate memory types that are suitable for non-blocking registration */
```

Reply:
👍🏼

Reply:
👍